llms.txt

```diff
@@ -12,54 +12,43 @@ The project is maintained by Timescale, a PostgreSQL database company.
 
 ## Installation
```

Comment:
The installation section is empty now?

Reply:
Because they moved to the extension dir. Good catch, fixed that also in the script.